includes/models/class-user.php

```diff
@@ -131,6 +131,11 @@ public function __construct( $args = [] ) {
 						'_model'    => 'attachment',
 						'_external' => true,
 					],
+
```

Comment:
This was already merged to master (verified badge), but I made a change to use the hp_email_verified meta instead and delete it when checked. Please sync this branch with master and resolve conflicts or create a new branch to move the valid code, in this case please delete this one.